apps/native-component-list/src/screens/Image/ImageFormatsScreen.tsx

```diff
@@ -84,16 +85,22 @@ const data: SectionListData<ImageSource>[] = [
       },
     ],
   },
-  {
```

Comment:
What if we do `Platform.OS === 'ios' && {` here and then `.filter(Boolean)` at the end of the `data` array?
It would be more clear in my opinion 😄